source/use_cases/aws-restaurant-management-demo/lib/lambda/kitchen-staff/complete-order/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
What's the goal of this? The CDK publishing process doesn't pay any attention to it, does it?